docs/hardening_guide.md

```diff
@@ -0,0 +1,405 @@
+# Hardening Guide
+
+This document provides prescriptive guidance for hardening a production installation of RKE2. It outlines the configurations and controls required to address Kubernetes benchmark controls from the Center for Information Security (CIS).
+
+If RKE2 is run in CIS mode, RKE2 applies a restrictive NetworkPolicy and PodSecurityPolicy. The restrictice NetworkPolicy allows for only namespace traffic wit the exception of DNS and applies to `kube-system`, `kube-public`, and `default` namespaces. The restrictive PodSecurityPolicy addresses CIS controls defined in section 5.2. More details can be found below.
+
+## Overview
+
+This document provides prescriptive guidance for hardening a production installation of RKE2 with Kubernetes v1.18. It outlines the configurations required to address Kubernetes benchmark controls from the Center for Information Security (CIS).
```

Comment:
Nit: This is kind of repeating line 3? I'm not going to cry about it - just noticed it. Not a big concern.

Reply:
That's how it is in the orginal. I'm happy to remove it here though.

Reply:
It's a nit so I'll leave it to you. It's not a big deal just reading it felt weird because it repeats.